### DISCUSSION TITLE
[LitmusChaos] Develop a terraform provider or scripts to provision litmuschaos functionalities

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#615] [Google Summer of Code 2022]
[LitmusChaos](https://litmuschaos.io/) is an open-source Chaos Engineering platform that enables teams to identify weaknesses & potential outages in infrastructures by inducing chaos tests in a controlled way. This project aims to develop a terraform provider or scripts to provision litmuschaos functionalities.

Mentor(s): Vedant Shrotria (@Jonsy13), Raj Babu Das (@rajdas98), Adarsh Kumar (@Adarshkumar14)

https://github.com/cncf/mentoring/blob/main/summerofcode/2022.md#develop-a-terraform-provider-or-scripts-to-provision-litmuschaos-functionalities

**Comment by Mostafa-wael** [comment]
I am really interested in this project. How can I apply for it as a part of my GSOC application? Is there any pre-steps?

**Reply by nate-double-u** [reply]
Hi @Mostafa-wael, check out the [GSoC Contributor/Student Guide](http://google.github.io/gsocguides/student/), and I think the [Finding the Right Project](https://google.github.io/gsocguides/student/finding-the-right-project) section especially. 

Ask questions! This thread is for potential contributors to chat with the mentors and get a clearer understanding of the projects and their needs.
